src/NuGet.Clients/NuGet.VisualStudio.Internal.Contracts/INuGetProjectManagerService.cs

```diff
@@ -46,7 +46,7 @@ public interface INuGetProjectManagerService : IDisposable
             CancellationToken cancellationToken);
 
         ValueTask<IReadOnlyList<ProjectAction>> GetUninstallActionsAsync(
-            string projectId,
+            IReadOnlyCollection<string> projectIds,
```

Comment:
This is an API break that'd break Codespaces support. 

cc @dtivel @rrelyea for review here. 

We probably need a PublicAPI file here too to help us manage this. 
cc @zivkan

Reply:
https://github.com/NuGet/NuGet.Client/pull/3708

Reply:
@nkolev92  I created threat on Codespace channel and tagged them. 
If it's the case then I can remedy this by create new separate api signature here.

Reply:
I talked with @rrelyea and [this break is ok](https://teams.microsoft.com/l/message/19:f5bb1554b1004a13897f486520d561a8@thread.skype/1602611417102?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=ad34ac55-6785-4e7c-82c3-7ba405d30c3c&parentMessageId=1602028186751&teamName=NuGet&channelName=NuGet%20for%20VS.Oe&createdTime=1602611417102).